src/main/java/ch/njol/skript/expressions/ExprTimeLived.java

```diff
@@ -0,0 +1,78 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Example;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.util.Timespan;
+import ch.njol.skript.util.Timespan.TimePeriod;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.entity.Entity;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Time Lived of Entity")
+@Description("""
+	Returns the total amount of time the entity has lived.
+	Note: This does not reset when a player dies.
+	""")
+@Example("""
+	clear all entities where [input's time lived > 1 hour]
+	""")
```

Comment:
```suggestion
@Example("clear all entities where [input's time lived > 1 hour]")
```
Since this example only takes up one line, you don't need the quote block